content/en/docs/setup/independent/setup-ha-etcd-with-kubeadm.md

```diff
@@ -46,7 +46,7 @@ this example.
 
 1. Configure the kubelet to be a service manager for etcd.
 
-    Running etcd is simpler than running kubernetes so you must override the
+    Since etcd was created first, we need to override the service priority so you must override the
```

Comment:
please change to:

> You must override the service priority by creating a new unit file that has higher precedence than the kubeadm-provided kubelet unit file.

this seems better to me.

Reply:
@neolit123 thanks, updated